prime-router/docs/api/api-keys.yml

```diff
@@ -0,0 +1,99 @@
+openapi: 3.0.2
+info:
+  title: Prime ReportStream
+  description: A router of public health data from multiple senders and receivers
+  contact:
+    name: USDS at Centers for Disease Control and Prevention
+    url: https://reportstream.cdc.gov
+    email: reportstream@cdc.gov
+  version: 0.2.0-oas3
+paths:
+  /settings/organizations/{organizationName}/public-keys:
+    get:
+      description: Retrieves all of the API public keys for the organization
+      security:
+        - OAuth2: [org_admin, system_admin]
+      parameters:
+        - in: path
+          schema:
+            type: string
+          required: true
+          name: organizationName
+          description: The name of the organization to fetch keys for
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ApiKeysResponse'
+        '401':
+          description: Unauthorized
+        '404':
+          description: Not Found
+    post:
+      description: Adds or overwrites a key for a specific scope.  Currently, is only implemented for orgName.*.report scope
+      security:
+        - OAuth2: [ system_admin ]
+      parameters:
+        - in: path
+          required: true
+          name: organizationName
+          description: The name of the organization to fetch keys for
+          schema:
+            type: string
+        - in: query
+          name: scope
+          required: true
+          description: The scope to add the key for
+          schema:
+            type: string
+        - in: query
+          name: kid
+          required: true
+          description: The kid that will be stored with the public key
+          schema:
+            type: string
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ApiKeysResponse'
+        '401':
+          description: Unauthorized
+        '404':
+          description: Not Found
+        '400':
+          description: Bad Request
+
+
+components:
+  securitySchemes:
+    OAuth2:
+      $ref: 'https://raw.githubusercontent.com/CDCgov/prime-reportstream/master/prime-router/docs/api/components/security_schemes.yml#/OAuth2'
+  schemas:
+    ApiKeysResponse:
+      description: Fetched or modified API Keys
+      type: object
+      properties:
+        orgName:
+          description: The name of the organization
+          type: string
+          example: simple_report
+        keys:
+          description: The list of keys for the organization
+          type: array
+          items:
+            description: A serialized public key
+            type: object
+            properties:
+              kty:
```

Comment:
Can you add descriptions and/or examples to the four fields?

Reply:
Done!